src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -865,8 +866,8 @@ public void AllWellKnownTypesBeforeCSharp7()
                 Assert.True(type <= WellKnownType.CSharp7Sentinel);
             }
 
-            // There were 204 well-known types prior to CSharp7
-            Assert.Equal(204, (int)(WellKnownType.CSharp7Sentinel - WellKnownType.First));
+            // There were 205 well-known types prior to CSharp7
```

Comment:
Is this comment correct?

Reply:
We should change this assertion to check `CSharp7Sentinel - First` instead. Then it remains true when adding new special types.
The point of `CSharp7Sentinel` is that our rules for finding well-known types changed a bit in C# 7 with regards to dealing with ambiguities:
```C#
                    // well-known types introduced before CSharp7 allow lookup ambiguity and report a warning
                    DiagnosticBag legacyWarnings = (type <= WellKnownType.CSharp7Sentinel) ? warnings : null;
```

---
In reply to: [298759023](https://github.com/dotnet/roslyn/pull/36722#discussion_r298759023) [](ancestors = 298759023)

Reply:
Should this new well-known type stay before the CSharp7Sentinel?

---
In reply to: [298769217](https://github.com/dotnet/roslyn/pull/36722#discussion_r298769217) [](ancestors = 298769217,298759023)

Reply:
after offline discussion it sounds like this should actually go after the sentinel so that we disallow multiple definitions of the type.

---
In reply to: [298770457](https://github.com/dotnet/roslyn/pull/36722#discussion_r298770457) [](ancestors = 298770457,298769217,298759023)